py/desitarget/cuts.py

```diff
@@ -182,7 +182,7 @@ def isGAIA_STD(ra=None, dec=None, galb=None, gaiaaen=None, pmra=None, pmdec=None
                parallax=None, parallaxovererror=None, ebv=None, gaiabprpfactor=None,
                gaiasigma5dmax=None, gaiagmag=None, gaiabmag=None, gaiarmag=None,
                gaiadupsource=None, gaiaparamssolved=None,
-               primary=None, test=False, nside=2):
+               primary=None, test=False, nside=2, dr="dr2"):
```

Comment:
Consider reverting all changes in `isGAIA_STD()`.

Reply:
Reverted all changes here too.